### PR TITLE
Django 3.0 support

### LINKED ISSUE #2352: Django 3.0 support
## Django 3.0 support?

[//]: # (What's it you're proposing? How should it be implemented?)
when will django 3.0 be supported?

 

[//]: # (Why should this feature be implemented?)

 

[//]: # ("Better to see something once than to hear about it a thousand times.")

### PR DESCRIPTION
**Pull request by browniebroke**
## Description

Upgrade to Django 3.0

- [x] Bump version in requirements & get build to pass
- [x] Update readme & documentation
- [x] Deploy to prod & do sanity testing 

## Rationale

Fixes #2352

## To test it out

You can try this out by running this command:

```
cookiecutter https://github.com/pydanny/cookiecutter-django --checkout django-30
```

Any help would be appreciated to help spot possible production issues. Please test it out and report any issues.

### CODE REVIEW

**Comment by sfdye** [comment]
Are we dropping support for 2.2 LTS? It's still supported until 2022:

https://www.djangoproject.com/download/

**Comment by browniebroke** [comment]
I believe the aim is to be on the latest version, we've never supported 2 Django versions at once, as far as I know.

File: {{cookiecutter.project_slug}}/config/settings/base.py [hunk 1]
```diff
@@ -226,7 +226,7 @@
 EMAIL_BACKEND = env(
     "DJANGO_EMAIL_BACKEND", default="django.core.mail.backends.smtp.EmailBackend"
 )
-# https://docs.djangoproject.com/en/2.2/ref/settings/#email-timeout
+# https://docs.djangoproject.com/en/dev/ref/settings/#email-timeout
```
Comment (sfdye) [inline {{cookiecutter.project_slug}}/config/settings/base.py]: 3.0?

Comment (browniebroke) [reply {{cookiecutter.project_slug}}/config/settings/base.py]: All our links are to the dev version so this is to make this link consistent with the rest. We should probably link to the stable one, but I'd say it's a separate issue. What do you think?

**Comment by AkashV10** [comment]
When will this be merged?

**Comment by sfdye** [comment]
After more testing, yes

**Comment by browniebroke** [comment]
@AkashV10 did you gave this a try? Would appreciate any feedback, good or bad. I've left the command you need to run in the description at the top.

**Comment by AkashV10** [comment]
@browniebroke no. I was getting errors with setuptools on Windows with existing version so I just used the djangoX repo instead and modified as needed.

**Pull request merged**